waspc/cli/src/Wasp/Cli/Command/CreateNewProject.hs

```diff
@@ -3,185 +3,64 @@ module Wasp.Cli.Command.CreateNewProject
-        ]
-
-createProjectFromTemplate :: Path System Abs (Dir WaspProjectDir) -> ProjectInfo -> Command ()
-createProjectFromTemplate absWaspProjectDir ProjectInfo {_appName = appName, _projectName = projectName, _templateName = maybeTemplateName} = do
-  cliSendMessageC $ Msg.Start "Creating project from template..."
-
-  templatePath <- getPathToRemoteTemplate maybeTemplateName
-
-  let projectDir = projectName
-
-  fetchTemplate templatePath projectDir
-  ensureTemplateWasFetched
-
-  replacePlaceholdersInWaspFile
-  where
-    getPathToRemoteTemplate :: Maybe String -> Command String
-    getPathToRemoteTemplate = \case
-      Just templateName -> return $ waspTemplatesRepo ++ "/" ++ templateName
-      Nothing -> throwProjectCreationError "Template name is not provided."
-      where
-        -- gh: prefix means Github repo
-        waspTemplatesRepo = "gh:wasp-lang/starters"
-
-    fetchTemplate :: String -> String -> Command ()
-    fetchTemplate templatePath projectDir = do
-      liftIO (try executeCmd) >>= \case
-        Left (e :: SomeException) -> throwProjectCreationError $ "Failed to create project from template: " ++ show e
-        Right _ -> return ()
-      where
-        executeCmd = callCommand $ unwords command
-        command = ["npx", "giget@latest", templatePath, projectDir]
-
-    -- gitget doesn't fail if the template dir doesn't exist in the repo, so we need to check if the directory exists.
-    ensureTemplateWasFetched :: Command ()
-    ensureTemplateWasFetched =
-      whenM (liftIO $ IOUtil.isDirectoryEmpty absWaspProjectDir) $
-        throwProjectCreationError "Are you sure that the template exists? 🤔 Check the list of templates here: https://github.com/wasp-lang/starters"
-
-    replacePlaceholdersInWaspFile :: Command ()
-    replacePlaceholdersInWaspFile = liftIO $ do
-      mainWaspFileContent <- IOUtil.readFileStrict absMainWaspFile
-
-      let replacedContent =
-            foldl
-              (\acc (placeholder, value) -> T.replace (T.pack placeholder) (T.pack value) acc)
-              mainWaspFileContent
-              replacements
-
-      IOUtil.writeFileFromText absMainWaspFile replacedContent
-      where
-        absMainWaspFile = absWaspProjectDir </> [relfile|main.wasp|]
-        replacements =
-          [ ("__waspAppName__", appName),
-            ("__waspProjectName__", projectName),
-            ("__waspVersion__", waspVersionBounds)
-          ]
-
-waspVersionBounds :: String
-waspVersionBounds = show (SV.backwardsCompatibleWith WV.waspVersion)
-
-throwProjectCreationError :: String -> Command a
-throwProjectCreationError = throwError . CommandError "Project creation failed"
+    printGettingStartedInstructions :: NewProjectName -> IO ()
+    printGettingStartedInstructions (NewProjectName projectName) = do
+      let projectFolder = projectName
+{- ORMOLU_DISABLE -}
+      putStrLn $ Term.applyStyles [Term.Green] $ "Created new Wasp app in ./" ++ projectFolder ++ " directory!"
```

Comment:
> You could just use basename to get the name of the dir, but then you are assuming that project dir is in current directory :D, which is not such a terrible assumption, but it is still an assumption.

Since we are constructing the path to the project from a `folderName` and prepending the `cwd` to it, I would say it holds.

It feels a bit weird to pass in the `projectName` inside and get the `abs path` out. Which we then parse to get the relative dir to the `cwd` but ... we know how we constructed the `abs path`.

Alternatively, we could return the `abs path` + `relative path` (or something like `projectDirInCwd`) in `ProjectDescription` 🤷

Reply:
I've used `basename` to get the folder name from the abs path and wrote down the assumption above the function definition. 

I did implement the "get relative path to folder from cwd" but I didn't like how it all looked like. So I went with the simplest option IMHO.